frontend/src/bundles/common/components/radio-card/radio-card.tsx

```diff
@@ -0,0 +1,48 @@
+import {
+    type Control,
+    type FieldPath,
+    type FieldValues,
+} from 'react-hook-form';
+
+import { useFormController } from '~/bundles/common/hooks/hooks.js';
+
+import styles from './styles.module.css';
+
+type Properties<T extends FieldValues> = {
+    name: FieldPath<T>;
+    id: string;
+    label: string;
+    value: string;
+    control: Control<T>;
+    checked?: boolean;
```

Comment:
I guess we don't use this property in this component

Reply:
@anton-otroshchenko Which one?

Reply:
checked because I did not find the usage of it in the component